src/plugins/v-focus/directives/focus-item.ts

```diff
@@ -1,17 +1,19 @@
 import { DirectiveBinding } from 'vue';
 import state from '../state';
 
+const bemBlock = 'v-focus-item';
+
 /**
  * Sets or removes a class depending on the current focus state.
  */
-function update(el: HTMLElement, binding: DirectiveBinding): void {
-  if (binding.value) {
-    el.classList.add('focus-item');
-  } else {
-    el.classList.remove('focus-item');
-  }
-
-  state.enabled = !!binding.value;
+function updated(el: HTMLElement, binding: DirectiveBinding): void {
+  const focusModifier = `${bemBlock}--focus`;
+  const enabled = !!binding.value;
+
+  el.classList.toggle(focusModifier, enabled);
```

Comment:
suggestion: If the item with the `v-focus-item` directive has no additional class, consider removing the empty `class` attribute from the DOM. It's a personal feeling, and while it may not bring any performance improvements, having an empty class attribute after removing `v-focus-item--focus` feels a bit wrong to me.

`
  if (enabled) {
    el.classList.toggle(focusModifier, enabled);
  } else if (el.classList.length <= 0) {
    el.removeAttribute('class');
  }
`

Reply:
I personally think this is a bit over engineered. I will keep as is.